bsp/nuvoton/libraries/nu_packages/Demo/smp_demo.c

```diff
@@ -150,25 +150,27 @@ static int go_happy_mutex(void)
 {
     rt_thread_t thread;
     rt_mutex_t  sem = rt_mutex_create("mutexsem", RT_IPC_FLAG_PRIO);
-
-    thread = rt_thread_create("mutex0", happy_mutex, (void *)sem, 2048, 25, 20);
-    if (thread != RT_NULL)
+    
+    if(sem != RT_NULL)
```

Comment:
这样一层层嵌套不优雅，如果条件不满足 直接退出就可以了

```suggestion
    if(sem == RT_NULL)
    {
        LOG_E();
        return (int)-RT_ERROR;
    }
```